package main

func main() {
	//errors 是一个接口

	//致命错误，程序会终止

	//recover 遵循先进后出的压栈调用模式
	//recover 只能用在defer中
}
